### ISSUE TITLE
2.1.4 and error mingw/msys2

### ISSUE THREAD

**Issue by 3rav** [microsoft/mimalloc#884]
I have this error with 2.1.4 (2.1.2 was OK)
```
 -- Library base name: mimalloc
  -- Version          : 2.1
  -- Build type       : release
  -- C Compiler       : D:/M/msys64/mingw64/bin/gcc.exe
  -- Compiler flags   : -Wall;-Wextra;-Wno-unknown-pragmas;-fvisibility=hidden;-Wstrict-prototypes;-ftls-model=initial-exec;-fno-builtin-malloc
  -- Compiler defines : 
  -- Link libraries   : psapi;shell32;user32;advapi32;bcrypt
  -- Build targets    : shared;static
  -- 
  -- Configuring done (0.9s)
  -- Generating done (0.0s)
  -- Build files have been written to: C:/_/B/src/build-MINGW64
  ninja: error: 'C:/_/B/src/mimalloc-2.1.4/bin/mimalloc-redirect.lib', needed by 'libmimalloc.dll', missing and no known rule to make it
  ==> ERROR: A failure occurred in build().
      Aborting...
  ==> Removing installed dependencies...
  checking dependencies...
```

**Comment by daanx** [comment]
ah, that is strange -- I don't think anything changed here with cmake?
Maybe try to use `make` instead of `ninja` to build?

**Comment by AlifianK** [comment]
`v2.1.4` tag doesn't contain `bin` folder, which is present in `v2.1.2`. If you look in the `bin` folder, you'll see that it contains the required `.lib` file.
@daanx perhaps you forget to include that?

**Comment by daanx** [comment]
Mmm, very strange -- maybe a merge error. Thanks for reporting this, I'll fix it with the release update later this week.

**Comment by daanx** [comment]
This is fixed now with v2.1.7. (it was an error in the .gitattributes)
